parser_genomes.py

```diff
@@ -69,11 +70,41 @@ def setup_directories(family, sample_list, filepath, step):
     if len(ped) > 1:
         print(f"Multiple ped files found: {ped}. Exiting!")
         exit()
+    if len(ped) == 0:
+        print(f"No ped files found: {ped}. Exiting!")
```

Comment:
Message should be something like f"No ped files found for family: {family}." Shouldn't say 'Exiting' because we're not exiting if the pedigree file does not exist.